### PR TITLE
Homepage content

### PR DESCRIPTION
**Pull request by abhikb2005**

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/130#" title="Author self-approved">abhikb2005</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by abhikb2005** [comment]
Hi @codrin-iftimie, resolving conflicts required me to choose changes for a couple of non-content files:

1. package 2.lerna_backup (probably not needed; it has been added automatically)
2. packages/website/gatsby-node.js (please check the changes into this file)

File: packages/docs/content/00-index.mdx [hunk 1]
```diff
@@ -100,12 +100,12 @@ import DocUpdatesSection from "@librarium/shared/src/components/common/DocUpdate
     {
       title: "Azure",
       cloud: "azure",
-      href: "/clusters?clusterType=azure_cluster#creatingclusters",
+      href: "/clusters#creatingclusters",
```
Comment (codrin-iftimie) [inline packages/docs/content/00-index.mdx:103]: please revert this

File: packages/website/gatsby-node.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ const GRAPHQL = {
   '/': () => {
     return `
     {
-      latestUpdates: allMdx(filter: {fields: {isDocsPage: {eq: true}}}) {
+      latestUpdates: allMdx {
```
Comment (codrin-iftimie) [inline packages/website/gatsby-node.js:11]: please revert this

File: package 2.lerna_backup [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (codrin-iftimie) [inline package 2.lerna_backup:1]: this file shouldn't be here

**Comment by abhikb2005** [comment]
Please review

**Pull request merged**